src/pages/Question.tsx

```diff
@@ -40,79 +38,81 @@ const MOVIES = gql`
   }
 `
 
+const GET_SCORE = gql`
+  query($imdbId: String!, $selectedTitle: String!, $remainingSeconds: Int!) {
+    scoreTitleResponse(
+      imdbId: $imdbId
+      selectedTitle: $selectedTitle
+      remainingSeconds: $remainingSeconds
+    )
+  }
+`
+
 const Question = () => {
   const history = useHistory()
 
-  useStorage()
-
-  const { addPoints, removeLife, currentImdbId, timeRemaining } = useStore()
+  const {
+    addPoints,
+    removeLife,
+    currentImdbId,
+    timeRemaining,
+    setTimeRemaining,
+  } = useStore()
 
   const { loading, error, data } = useQuery<MovieData>(MOVIES, {
-    variables: { imdbId: currentImdbId },
+    fetchPolicy: 'cache-only',
```

Comment:
This causes Apollo to never do a network request. If the poster fetch has not been done before, this page will fail, as it does not have any data.